### DISCUSSION TITLE
[Bug?]: `corepack enable` installation method not working on Ubuntu 22.10

### DISCUSSION THREAD

**Discussion by JonathanLindsey** [yarnpkg/berry#5060] [Q&A] [answered]
### Self-service

- [ ] I'd be willing to implement a fix

### Describe the bug

On Ubuntu 22.10 (latest non-LTS) with node v18.7.0 yarn cannot be installed [as documented](https://github.com/yarnpkg/berry/blob/1756c0158bed98ec96430f2a1ec6e3a12ebbd2b8/packages/gatsby/content/getting-started/install.md) with `corepack enable`. Instead I get `corepack: command not found`.

I am able to install corepack but it also requires sudo which is not documented: `sudo npm i -g corepack`. Is sudo not in the documentation in order to be OS agnostic?

### To reproduce

1. Start with a fresh Ubuntu 22.10 installation.
2. `sudo apt install npm` to install nodejs v18.7.0.
3. `corepack enable`

### Environment

After successfully installing yarn with `sudo npm i -g corepack`:

```shell
System:
  OS: Linux 5.19 Ubuntu 22.10 22.10 (Kinetic Kudu)
  CPU: (1) x64 Intel(R) Core(TM) i5-8265U CPU @ 1.60GHz
Binaries:
  Node: 18.7.0 - /tmp/xfs-433908c4/node
  Yarn: 1.22.19 - /tmp/xfs-433908c4/yarn
  npm: 8.18.0 - /usr/bin/npm
```
This is a Virtual Box VM running under Pop!_OS 22.04 LTS

### Additional context

Is this a documentation issue, yarn issue or something specific with Ubuntu?

**Comment by RDIL** [comment, marked as answer]
You have installed Node through a system package manager, which doesn’t include corepack.  You need to either install corepack first (`npm i -g corepack`) or use a different method of installing Node, such as nvm.

This is just a case of the apt package not shipping everything that the typical Node.js distributions include.

As for requiring sudo, it’s just a permission issue.  Typically, you should have control over the global package store.
